docs/changehistory/NextVersion.md

```diff
@@ -3,10 +3,50 @@
 Table of contents:
 
 - [@itwin/appui-react](#itwinappui-react)
+  - [Deprecations](#deprecations)
   - [Changes](#changes)
 
 ## @itwin/appui-react
 
+### Deprecations
+
+- Deprecated `SignInProps` instead use `React.ComponentProps<typeof SignIn>`. [#991](https://github.com/iTwin/appui/pull/991)
+- Deprecated `AccuDrawDialogProps` instead use `React.ComponentProps<typeof AccuDrawDialog>`. [#991](https://github.com/iTwin/appui/pull/991)
```

Comment:
Let's add one code snippet example as well.

Reply:
Is one snippet enough to summarize all changes. Or a code snippet for each type of props?

Reply:
Added a code snippet to every package group.